Source/santad/EventProviders/SNTEndpointSecurityManager.h

```diff
@@ -20,7 +20,7 @@
 #include <EndpointSecurity/EndpointSecurity.h>
 
 // Gleaned from https://opensource.apple.com/source/xnu/xnu-4903.241.1/bsd/sys/proc_internal.h
-const pid_t PID_MAX = 99999;
+#define PID_MAX 99999
```

Comment:
What's the reason for this change?

Reply:
This was originally because SNTEndpointSecurityManager was getting linked multiple times, but it doesn't look like that's a problem anymore. I'll revert this.